File: 02 Ad-Hoc/3432 Interceptando Informações/3432.cpp
#include <iostream>

using namespace std;

int main(){
  int inp;
  bool falha = false;
  for(int i = 0; i < 8; i++){
    cin >> inp;
    if (inp == 9){
      falha = true;
      break;
    }
  }
  if (falha) cout << "F\n";
  else cout << "S\n";
}

// https://www.beecrowd.com.br/judge/pt/problems/view/3432
// Ranking 00015º, Time 0.000, Submiss Date 29/05/2023 03:25:46 (C++17)

File: 02 Ad-Hoc/3432 Interceptando Informações/3432.rb
falha = false
inputs = gets.chomp.split.map(&:to_i)

inputs.each do |inp|
  if inp == 9
    falha = true
    break
  end
end

if falha
  puts "F"
else
  puts "S"
end

# https://www.beecrowd.com.br/judge/pt/problems/view/3432
# Ranking 00001º, Time 0.149, Submiss Date 29/05/2023 03:32:21 (Ruby 2.7)